examples/03-eaa-diploma/specification.yml

```diff
@@ -0,0 +1,57 @@
+user_claims:
+  binding:
```

Comment:
I expect this to be the equievalent of cnf for claim-based binding. This should be a standardized claim in the spec imo.